koku/cost_models/serializers.py

```diff
@@ -96,20 +101,31 @@ def validate(self, data):
             return data
 
 
+def get_json(path):
+    """Obtain API JSON data from file path."""
+    json_data = None
+    with open(path) as json_file:
+        try:
+            json_data = json.load(json_file)
+        except (IOError, json.JSONDecodeError) as exc:
+            LOG.exception(exc)
+    return json_data
+
+
 class RateSerializer(serializers.Serializer):
     """Rate Serializer."""
 
     DECIMALS = ("value", "usage_start", "usage_end")
     RATE_TYPES = ("tiered_rates",)
 
     metric = serializers.DictField(required=True)
-    cost_type = serializers.ChoiceField(choices=CostModelMetricsMap.COST_TYPE_CHOICES)
+    cost_type = serializers.ChoiceField(choices=constants.COST_TYPE_CHOICES)
     tiered_rates = serializers.ListField(required=False)
 
     @property
     def metric_map(self):
         """Return a metric map dictionary with default values."""
-        metrics = CostModelMetricsMap.objects.values("metric", "default_cost_type")
+        metrics = get_json(os.path.join(BASE_DIR, "api/metrics/data/cost_models_metric_map.json"))
```

Comment:
You've already got a constant defined in another file for this location, I'd use that.

Reply:
updated.